src/watchman/models.clj

```diff
@@ -206,12 +218,38 @@
     (k/set-fields fields)
     (k/where {:id id})))
 
+(defn snooze-msg [snooze-until]
```

Comment:
This doesn't seem like it belongs in `models`, since it is presentation logic and doesn't touch the models.